Xamarin.Forms.Core/Shapes/Ellipse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms.Shapes
+{
+    [RenderWith(typeof(_EllipseRenderer))]
+    public sealed class Ellipse : Shape
+    {
+		public Ellipse()
+		{
+			Aspect = Stretch.Fill;
```

Comment:
you're not setting a default property here, but a strongly set one, preventing styling. Use defaultValueCreator for that